compiler/rustc_span/src/symbol.rs

```diff
@@ -1347,6 +1347,10 @@ symbols! {
         store,
         str,
         str_alloc,
+        str_split_whitespace,
+        str_trim,
+        str_trim_start,
+        str_trim_end,
```

Comment:
```suggestion
        str_trim_end,
        str_trim_start,
```

Reply:
Please squash this change in your existing commit.

Reply:
Yeah, I should learn the alphabet